01-introduction.Rmd

```diff
@@ -2,68 +2,65 @@
+This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path clearer and less scary.
 
-Whether you're a data scientist using your skills in an education job or an educator who wants to learn data science skills, we invite you to read this book and put these techniques to work in the real world. We think that your work in the education community will help decide how education and data science come together going forward.
+Whether you're a data scientist working in education or an educator wanting to learn data science, we invite you to read this book and put these techniques to work in the real world. Your work in the education community will help decide how education and data science come together.
 
 ## Learning data science in education
 
-Over the coming chapters, we'll be learning together about what data science in education can look like. But to understand why we were compelled to write about the topic, we need to talk about why data science in education is not such a straightforward thing. 
-
-Learning data science in education is challenging because there isn't a universal vision for that role yet. Data science in education isn't straightforward because the role itself is not straightforward. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. 
+In the coming chapters, you'll explore examples of data science in education. But first, let's talk about why data science in education is not easily definable. 
 
-This imaginary building we call education also has rooms most residents never see---rooms where business and finance staff plan the most efficient use of limited funds. The transportation department plans bus routes across vast spaces. University administrators search for the best way to measure career readiness. Education consultants study how students perform on course work and even how they feel about class materials. 
+Being a data scientist in education is challenging because there is not a universal vision of what that means. For starters, education as a field is complex. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. Students start school, leave school, finish school, and return to school. 
```

Comment:
```suggestion
Being a data scientist in education is challenging because there is no universal vision of what that means. Education is a complex field. If education were a building, it would be multi-storied with countless rooms. There are privately and publicly funded schools with more than 18 possible grade levels. Students learn alone or with others in a classroom. Their journeys involve starting, leaving, finishing, and even returning to school. 
```